1.0.0_lint_schema.json

```diff
@@ -0,0 +1,176 @@
+{
+    "$schema": "https://reuse.software/schemas/1.0.0_lint_schema.json",
+    "$ref": "#/definitions/Result",
+    "definitions": {
+        "Result": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "lint_version": {
+                    "type": "string"
+                },
+                "reuse_spec_version": {
+                    "type": "string"
+                },
+                "reuse_tool_version": {
+                    "type": "string"
+                },
+                "files": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/File"
+                    }
+                },
+                "non_compliant": {
+                    "$ref": "#/definitions/NonCompliant"
+                },
+                "summary": {
+                    "$ref": "#/definitions/Summary"
+                }
+            },
+            "required": [
+                "files",
+                "lint_version",
+                "non_compliant",
+                "reuse_spec_version",
+                "reuse_tool_version",
+                "summary"
+            ],
+            "title": "Welcome"
+        },
+        "File": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "path": {
+                    "type": "string"
+                },
+                "copyrights": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Copyright"
+                    }
+                },
+                "licenses": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Copyright"
```

Comment:
Is the object name Copyright the correct name if it also used in a licensing context?

Reply:
Maybe Details or FindingDetails?